init.lua

```diff
@@ -202,8 +202,8 @@ return {
 
 		if action == "jump" then
 			ya.manager_emit("cd", { bookmarks[selected].desc })
-			ya.manager_emit("arrow", { -99999999 })
-			ya.manager_emit("arrow", { bookmarks[selected].cursor })
+			ya.manager_emit("arrow", { tostring(-99999999) })
+			ya.manager_emit("arrow", { tostring(bookmarks[selected].cursor) })
```

Comment:
```suggestion
			ya.manager_emit("arrow", { -99999999 })
			ya.manager_emit("arrow", { bookmarks[selected].cursor })
```

The [latest commit](https://github.com/sxyazi/yazi/commit/ff14b9a265e8ef09e0313277a1b306b0fe804fde) should include fixes for it, so dont need to add `tostring`